src/app/shared/modals/line-listing-add-modal/line-listing-add.component.scss

```diff
@@ -0,0 +1,45 @@
+@use '/src/variables.scss' as *;
+
+.mat-dialog-title {
+  padding: 40px 25px 25px;
+
+  > h1 {
+    margin: 0;
+    font-size: 28px;
+  }
+
+  .button-dialog-close {
+    top: 2px;
+    right: 2px;
+  }
+}
+
+.mat-dialog-content {
+  padding: 14px 16px;
+}
+
+.info-text {
+  color: map-get($sormas-accent, 500);
```

Comment:
We could use : mat.get-color-from-palette here as well as in the other scss file.